src/Microsoft.OData.Core/UriParser/Aggregation/ApplyClause.cs

```diff
@@ -102,6 +104,15 @@ internal List<string> GetLastAggregatedPropertyNames()
                     : CreateAggregatePropertiesUriSegment(aggregateExpressions);
             }
 
+            if (computeExpressions != null)
+            {
+                string computeProperties = string.Join(",", computeExpressions.Select(e => e.Alias));
+                if (!string.IsNullOrEmpty(computeProperties))
+                    result = string.IsNullOrEmpty(result)

```

Comment:
Even though this is a single line within the if-statement, we prefer to still have the { } based on our style guide.